app/controllers/api/v2/trends_controller.rb

```diff
@@ -45,7 +50,12 @@ def resource_scope_for_index(options = {})
       end
 
       def resource_scope(options = {})
-        @resource_scope ||= scope_for(Trend.types, options)
+        @resource_scope ||= scope_for(ForemanStatistics::Trend.types, options)
+      end
+
+      def ensure_statistics_plugin
+        plugin = Foreman::Plugin.find(:foreman_statistics).nil?
+        not_found(_('For access to /trends API you need to install Foreman Statistics plugin')) unless plugin
```

Comment:
I beleive it should be if instead of unless.

Reply:
right :facepalm: